pkg/controller/wip/controller_test.go

```diff
@@ -31,34 +32,57 @@ import (
 	"k8s.io/kubernetes/pkg/client/testing/core"
 )
 
-func TestReconcileBroker(t *testing.T) {
-	fakeKubeClient, fakeCatalogClient, fakeBrokerCatalog, _, _, testController, _, stopCh := newTestController(t)
+// broker used in most of the tests that need a broker
+var broker = &v1alpha1.Broker{
```

Comment:
I agree with this refactor, but prefer these as functions that return objects so that a test can't accidentally mutate one and clobber the other tests.